typed_ast/conversions.py

```diff
@@ -170,24 +170,34 @@ def visit_Ellipsis(self, n):
         return ast35.Index(ast35.Ellipsis(lineno=-1, col_offset=-1))
 
     def visit_arguments(self, n):
-        def convert_arg(arg):
+        def convert_arg(arg, annotation):
             if isinstance(arg, ast27.Name):
                 v = arg.id
             elif isinstance(arg, ast27.Tuple):
                 v = self.visit(arg)
             else:
                 raise RuntimeError("'{}' is not a valid argument.".format(ast27.dump(arg)))
-            return ast35.arg(v, None, lineno=arg.lineno, col_offset=arg.col_offset)
+            return ast35.arg(v, annotation, lineno=arg.lineno, col_offset=arg.col_offset)
 
-        args = [convert_arg(arg) for arg in n.args]
+        def get_type_comment(i):
+            if i < len(n.type_comments):
```

Comment:
Huh, will this work if there are no type comments -- do we end up with an empty sequence for `n.type_comments`, or `None`?